device_backends/LogicalNameMapping/src/LNMBackendVariableAccessor.cc

```diff
@@ -234,8 +234,8 @@ namespace ChimeraTK {
           this->buffer_2D[0][i] = userTypeToUserType<UserType>(vtEntry.latestValue[i + _wordOffsetInRegister]);
           this->_dataValidity = vtEntry.latestValidity;
         }
+        this->_versionNumber = vtEntry.latestVersion;
```

Comment:
question for my understanding: is it actually necessary for the new math-plugin behavior, to keep the latest version number even for poll-type acessors (on the parameters), or is this an independent improvement you just added here?

Reply:
It is essential, since otherwise the check in `MathPluginFormulaHelper::checkAllParametersWritten()` no longer works, because we now obtain even push-type parameters without wait_for_new_data. The check would always evaluate true and garbage values would be written to the device in the startup phase.

Reply:
I see. Can you please update the comment in MathPluginFormulaHelper::checkAllParametersWritten since it's no longer correct.

Reply:
done. I also added a comment here explaining that it is important for the math plugin and that it does not violate the spec.